swift/Workflow/Sources/OSLog+Workflow.swift

```diff
@@ -16,7 +16,82 @@
 
 import os.signpost
 
-extension OSLog {
+fileprivate extension OSLog {
     static let workflow = OSLog(subsystem: "com.squareup.Workflow", category: "Workflow")
     static let worker = OSLog(subsystem: "com.squareup.Workflow", category: "Worker")
 }
+
+// MARK: -
+
+final class WorkflowLogger {
+    // MARK: Workflows
+
+    static func logWorkflowStarted<WorkflowType>(ref: WorkflowNode<WorkflowType>) {
```

Comment:
this has the side effect of losing some information about the call site (I think `os_log` keeps track of `#file` and `#line`, but it seems like we don't really care about that so LGTM